### PR TITLE
Bug 1193162 - [JSDOC] write documentation of app-deck in jsdoc format

### PR DESCRIPTION
**Pull request by dwi2**
https://bugzilla.mozilla.org/show_bug.cgi?id=1193162

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0fceb349aab08bba5bfc9d1e95aa6e10819d5922

File: tv_apps/app-deck/README.md [hunk 1]
```diff
@@ -1,10 +1,28 @@
...
 1. displaying all installed app.
 2. uninstalling app
 3. pinning app to `Home`
 
+From JS point-of-view, App deck is composed of three classes:
+* AppDeck
+* ContextMenu
+* PromotionList
+
+AppDeck class is the main controller of this app. It handles all beha. The other two classes work as submodule of AppDeck and are both initialized by AppDeck. AppDeck also utilize other classes or module that help it do things.
```
Comment (rexboy7) [inline tv_apps/app-deck/README.md]: `behavior`?

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5ac319dbe2481e1638987b1c670161040003dcd3

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c1cd1259d4d65ac10427416c489493e60ab1c909

File: tv_apps/app-deck/js/context_menu.js [hunk 2]
```diff
@@ -4,6 +4,15 @@
 /* jshint nonew: false */
 
 (function(exports) {
+
+
+  /**
+   * It handles every thing related to contextmenu in app-deck.
```
Comment (rexboy7) [inline tv_apps/app-deck/js/context_menu.js]: `everything`?

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6863e74f3705d9754e14b3643c626bd99a809ac6

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=fb7eb7ef4708a248ac367cf44e49f3da97f840e1

**Pull request merged**